manifests/d/Docker/DockerDesktop/4.5.1/Docker.DockerDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.1.0.schema.json
+PackageIdentifier: Docker.DockerDesktop
+PackageVersion: 4.5.1
+PackageLocale: en-US
+Publisher: Docker Inc.
+PublisherUrl: https://www.docker.com
+PrivacyUrl: https://www.docker.com/legal/privacy
+Author: Docker Inc.
+PackageName: Docker Desktop
+PackageUrl: https://www.docker.com/products/docker-desktop
+License: Proprietary
+LicenseUrl: https://www.docker.com/legal/docker-subscription-service-agreement
+Copyright: Copyright © 2015-2021 Docker Inc. All rights reserved.
+CopyrightUrl: https://www.docker.com/legal/docker-subscription-service-agreement
+ShortDescription: Docker Desktop is an application for macOS and Windows machines for the building and sharing of containerized applications. Access Docker Desktop and follow the guided onboarding to build your first containerized application in minutes.
+Moniker: docker
+Tags:
+- container
+- containerization
+- virtualization
+ReleaseNotes: |
+  New
+  • Docker Desktop 4.5.0 introduces a new version of the Docker menu which creates a consistent user experience across all operating systems. For more information, see the blog post New Docker Menu & Improved Release Highlights with Docker Desktop 4.5
+  • The ‘docker version’ output now displays the version of Docker Desktop installed on the machine.
+
+  Upgrades
+  • Amazon ECR Credential Helper v0.6.0
+
+  Bug fixes and minor changes
+  • Increased the filesystem watch (inotify) limits by setting fs.inotify.max_user_watches=1048576 and fs.inotify.max_user_instances=8192 in Linux. Fixes docker/for-mac#6071.
+  • Fixed an issue related to compose app started with version 2, but the dashboard only deals with version 1
+  • Fixed an issue where Docker Desktop incorrectly prompted users to sign in after they quit Docker Desktop and start the application.
```

Comment:
```suggestion
  Bug fixes and minor changes
  • Fixed an issue that caused new installations to default to the Hyper-V backend instead of WSL 2.
  • Fixed a crash in the Docker Dashboard which would make the systray menu disappear.
```